### DISCUSSION TITLE
MCP 44 tools - 3 days - Built a high end culinary analytics dashboard for NYC and Chicago

### DISCUSSION THREAD

**Discussion by viscoussnake** [modelcontextprotocol/.github#58] [Show and tell]
### Pre-submission Checklist

- [X] I have verified that this discussion would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Discussion Topic

https://viscoussnake.github.io/CMNYCCHICAGO/

I can envision A LOT more with MCP.

# CulinaireMetrics
Premiere Culinary Intelligence Analytics

## Overview

CulinaireMetrics is an advanced analytics platform designed to provide insights into the culinary world, focusing on high-end restaurants in New York City. The application leverages various APIs to gather data, analyze trends, and enhance restaurant profiles with up-to-date information.

## Key Features

- **Real-Time Data Updates:** Regularly updates restaurant data using Brave Search and Google Places APIs.
- **Advanced Analysis:** Utilizes the Grok AI model to analyze restaurant data and provide insights on sentiment, innovation, momentum, and resilience.
- **Automated Scheduling:** Uses cron jobs to automate data updates and analysis at specified intervals.
- **Comprehensive Reporting:** Generates detailed reports on restaurant performance and trends.

## Technologies Used

- Node.js
- Brave Search API
- Google Places API
- XAI for advanced analytics
- GitHub Actions for CI/CD

**Comment by rublev** [comment]
This is awesome. Would love to chat about it, do you have an email I can hit you at? I don't have a reddit account but saw your posts there. Cheers.
